### ISSUE TITLE
UI decluttering by restructuring the Drawer

### ISSUE THREAD

**Issue by sonora** [osmandapp/Osmand#5466]
OsmAnd has a reputation to have a very complex user interface. When showing OsmAnd to new users, I often observe them being overwhelmed by the multitude of options in our Drawer menu, many of which they will not use until becoming much more advanced (some may never use, or some are setup features, and used only once).

Here is a suggestion to collapse the Drawer menu to some degree without affecting our overall richness of options and features. In a sense this would leave the top level Drawer menu more cleaned up, mostly showing "actions", while "Settings" would be hidden one layer behind: 

- "Configure map" could be moved to be a menu entry under "Settings", before "General"
- "OsmAnd Live" could be a tab under "Download maps"
- "Configure screen" could be moved to be a menu entry under "Settings", after "Configure map"
- "Plugins" could be moved to be a menu entry under "Settings", after "Navigation settings"
- "Builds" could be moved under the "Development Plugin's" settings

**Comment by ilyaguy** [comment]
Also, it is some confusing to have two menu items "Configure map" and "Configure screen".
I understand that they do different things, but sometimes it need to open both of them to find some option.

**Comment by sonora** [comment]
Yes, the logic is that that
- "Configure map" contains how **the map itself looks** in terms of map style, overlays, etc., while
- "Configure screen" determines **which app widgets** should be visible on the map screen.

Both menus are long enough already, so it seems not feasible to combine both .... Maybe rewording one or the other could help (e.g. use "Setup screen" for the latter), if anybody has an idea.  Or maybe putting the 2 headlines next to each other on the settings screen as I suggest here, rather than further apart as they are now.

**Comment by vshcherb** [comment]
I oppose it, removing items and putting them under nested structure will make it even less usable & understandable. 
We are not trying to make a clean menu for the sake of a clean menu. OsmAnd is complex because it has tons of features not because it has lots of items in Drawer, there is no need to hide Plugins cause it very self-explanatory where under Settings it is not reachable. 

I'm strongly against to change it without a reason cause majority of people get used to this structure.

P.S. Maps application are complex by itself. I don't want to make it more complex as it now but I don't want to make it less complex. I don't see adding more than 1-2 items in long term future and I definitely don't see a problem with 1.5-2 scroll in the drawer.

We can keep this discussion open.

**Comment by sonora** [comment]
Yes, I know, it is a tough discussion. Primarily the only simplification I can think of is distinguishing between
- **Category 'Actions'**, which are the basic functionality of the app and users potentially need it when operating the app on a daily basis, vs.
- **Category 'Settings'**, something more or less set one time (or rarely) in accordance with personal needs and preferences, but during daily operations usually left alone.

The vast majority of all simplification attemps I have suggested in recent years try to make classifications along those lines.

I do believe that 'Actions' need to be readily accessible for obvious reasons of ergonometry, while 'Settings' can be hidden in deeper hierarchies, both because advanced users will find them there if they need them, while good defaults will work for the less advanced user who never wants to edit or see them.

**Comment by vshcherb** [comment]
@sonora  That's why there is a delimiter between actions/views and settings. That's exactly the reason behind it.
But I don't see an issue having "Measure distance" as a separate item cause it obviously explains what is it.

OsmAnd Live is a separate because it maintains subscription, we hope in future subscription will have more valuable features and that's part will be reorganized. As of today we need to keep it separate.
